src/interfaces/proxies/actions/ICommonActions.sol

```diff
@@ -20,7 +20,7 @@ interface ICommonActions {
   /**
    * @notice Exits system coins from the safeEngine
    * @param  _coinJoin Address of the CoinJoin contract
-   * @param  _coinsToExit Amount of coins to exit [wad]
+   * @param  _coinsToExit Amount of coins to exit [wad] // REVIEW: _coinsToExit is treated as RAD!
    */
   function exitSystemCoins(address _coinJoin, uint256 _coinsToExit) external;
```

Comment:
it is! systemCoins are RAD.

Reply:
I think it may be misleading to input a `WAD` value at `joinSystemCoins()`, but a `RAD` value at `exitSystemCoins()`. Considering that both `coinJoin.join()` and `coinJoin.exit()` accept a `WAD` amount, it is even more confusing. What do you think?

Reply:
As the previous matter had most probably been an already addressed design decision, I assumed that what was wrong here was the documentation notes, which said `[wad]` rather than `[rad]`.

Reply:
hmmmm and we're doing the `/ RAY` inside, it's weird, but ok, good catch fixing the comment